transcom/mymove
# MB-16794 & MB-16679 add or modify content in various areas

## [16794 ticket](https://dp3.atlassian.net/browse/MB-16794?atlOrigin=eyJpIjoiMGVkZDc2MTQ5OWE2NDJmMzllNmIxZmZmMjJkOTI3MTQiLCJwIjoiaiJ9) & [16679 ticket](https://dp3.atlassian.net/browse/MB-16679?atlOrigin=eyJpIjoiNzY5YjQ4ODA0ZGRhNGJmYjgzMzViMjQ3M2VkYzE2MGMiLCJwIjoiaiJ9)

## Summary

Updates:

Ammend orders: new and revised content (in app and updated storybook comp too)

HHG Shipment Language change on the shipment move page 

On the Moving info page Move 101: You don't need to get the details perfect section... new and revised content

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1. Access the SM app
2. These are in various areas. You can Create new user and go through whole move, but you can also go in to various levels of test scenarios to see some of these (different types of non completed, unsubmitted moves for 2 of these, and combo@ppm.hhg is a good one for orders changes) if that's easier for you.
3. `make storybook `command will launch storybook to see the orders change (?path=/docs/customer-components-step--docs).

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
<img width="740" alt="HHG ship card" src="https://github.com/transcom/mymove/assets/108361430/f02bb3c2-ef3f-44c6-ad19-06fd1b4c948e">

<img width="988" alt="ammendedorder" src="https://github.com/transcom/mymove/assets/108361430/c1e1b4f7-e534-449c-b93a-d32467f50195">

In storybook:

<img width="1074" alt="storybookorders" src="https://github.com/transcom/mymove/assets/108361430/753229ff-6f03-4d7f-88ac-3ae775093c72">
